_posts/2019-04-09-announcing-riff-0-3-0.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: single
+title: "Announcing riff v0.3.0"
+header:
+  overlay_image: /images/frosty-morning.jpg
+excerpt:
+  With updated Knative components, Windows CLI, and a new Buildpacks design for Functions
+permalink: /blog/announcing-riff-0-3-0/
+---
+
+We are happy to announce the release of [riff v0.3.0](https://github.com/projectriff/riff/tree/v0.3.x). Thank you all  riff, Buildpacks, and Knative contributors.
+
+The riff CLI can be downloaded from our [releases page](https://github.com/projectriff/riff/releases/tag/v0.3.0) on GitHub. Please follow one of the [getting started](/docs) guides, to create a new cluster on GKE, Minikube, Docker Desktop for Mac, or Docker Desktop for Windows.
+
+Notable changes in this release include:
+- update to Buildpacks 0.1, progress towards user defined invokers/buildpacks
```

Comment:
we should mention that the Java buildpack brings in Java 11 so any Maven build POMs should be configured to use Java 8 or later

```
	<properties>
		<maven.compiler.source>1.8</maven.compiler.source>
		<maven.compiler.target>1.8</maven.compiler.target>
	</properties>
```